### PR TITLE
30s down-moratorium before allowing suspension

### PR DESCRIPTION
**Pull request by hakonhall**
If all services of a node are down, we used allow suspension.  If those services are monitored with /state/v1/health, we have the timestamp the service became unhealthy - the "since" timestamp.  Now we will require the services to have been down for at least 30s before allowing suspension based on unhealthiness.

Also, for config servers only, we will log all healthiness transitions to track down some orchestrator issues.

### CODE REVIEW

File: application-model/src/main/java/com/yahoo/vespa/applicationmodel/ClusterId.java [hunk 1]
```diff
@@ -12,6 +12,9 @@
 
     public static final ClusterId CONFIG_SERVER = new ClusterId("zone-config-servers");
     public static final ClusterId CONTROLLER = new ClusterId("controller");
+    public static final ClusterId CONFIG_SERVER_HOST = new ClusterId("configserver-host");
+    public static final ClusterId CONTROLLER_HOST = new ClusterId("configserver-host");
```
Comment (hmusum) [inline application-model/src/main/java/com/yahoo/vespa/applicationmodel/ClusterId.java:16]:
```suggestion
    public static final ClusterId CONTROLLER_HOST = new ClusterId("controller-host");
```

**Review by hmusum — CHANGES_REQUESTED** [review]
One suggested change, otherwise LGTM

File: application-model/src/main/java/com/yahoo/vespa/applicationmodel/ClusterId.java (hunk 1 above)
Comment (hakonhall) [reply application-model/src/main/java/com/yahoo/vespa/applicationmodel/ClusterId.java:16]: Thank you!

**Review by hmusum — APPROVED** [review]

**Comment by hakonhall** [comment]
I'm unable to reproduce the Travis failure (ControllerTest, testDevDeployment), and I no longer have the ability to trigger another run on Travis.  I'll merge and make a revert just in case.

**Pull request merged**